gson/src/main/java/com/google/gson/internal/UnsafeAllocator.java

```diff
@@ -31,6 +32,9 @@
 public abstract class UnsafeAllocator {
   public abstract <T> T newInstance(Class<T> c) throws Exception;
 
+  @CFComment({"possible missing annotation in jdk for class Field, method get as the doc says it can take null argument #1",
+  "class Method, function invoke is annotated to receive non-null args but CFComment says it might permit null #2"})
+  @SuppressWarnings({"nullness:argument.type.incompatible","unboxing.of.nullable"})
```

Comment:
Seems like more annotations need to be added in the jdk. Remove the comment after you do so.